pages/osx/tree.md

```diff
@@ -22,10 +22,14 @@
 
 `tree -s -h --du`
 
-- Find files within the tree hierarchy, using a wildcard (glob) pattern, and pruning out directories that don't contain matching files:
+- Print files within the tree hierarchy, using a wildcard (glob) pattern, and pruning out directories that don't contain matching files:
 
 `tree -P '{{*.txt}}' --prune`
 
-- Find directories within the tree hierarchy, pruning out directories that aren't ancestors of the wanted one:
+- Print directories within the tree hierarchy, using the wildcard (glob) pattern, and pruning out directories that aren't ancestors of the wanted one:
 
 `tree -P {{directory_name}} --matchdirs --prune`
+
+- Print the tree ignoring the given directories:
+
+`tree -I 'vendor|.git|node_modules'`
```

Comment:
Is this a legal syntax? Or it's just one directory at the time? Anyway, it should be inside `{{}}`.

(Is `vendor|.git|node_modules` legal, or it should be `{{vendor|.git|node_modules}}` instead)?

Reply:
It should be `{{vendor|.git|node_modules}}` since it is a parameter as any other.

Reply:
Maybe it is, but I don't know the specifics of this command. That's why I asked it in the first place.

Reply:
It's legal syntax, it means to ignore any name that is `vendor` OR `.git` OR `node_modules`.